hnn_core/params.py

```diff
@@ -47,6 +48,46 @@ def __init__(self, params_fname):
                 params[key] = params_input.pop(key)
             self[key] = params[key]
 
+    def read_param_file(self, params_fname):
```

Comment:
I would have these two methods as functions from which you initialize the `Params` object.

Also then you would add it to api.rst

Reply:
So the user has to call create_param_from_json() or create_param_from_param()? Which internally creates a Param instance that they can manipulate?